ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -134,7 +134,7 @@ export class ImportCommand extends IronfishCommand {
         }
       }
 
-      if (typeof data.createdAt === 'string') {
+      if (data.version === 1) {
         data.createdAt = null
```

Comment:
This is optional but it will allow any migrations later to work fully. We've fully upgraded it here so we can bump the version.

```suggestion
        data.createdAt = null
        data.version = 2
```